packages/in_app_purchase/in_app_purchase_storekit/lib/src/store_kit_wrappers/sk_payment_queue_wrapper.dart

```diff
@@ -405,7 +405,8 @@ class SKPaymentWrapper {
       'applicationUsername': applicationUsername,
       'requestData': requestData,
       'quantity': quantity,
-      'simulatesAskToBuyInSandbox': simulatesAskToBuyInSandbox
+      'simulatesAskToBuyInSandbox': simulatesAskToBuyInSandbox,
+      'paymentDiscount': paymentDiscount?.toMap(),
```

Comment:
We should have some test for the `toMap` method, can we update the test to include the discount? If we don't have an existing test, do you mind adding a test for the `toMap` method?

Reply:
Done. I've added a new test checking the values of the `SKPaymentWrapper` object and the result of the `toMap` method explicitly.